modules/plink2/extract/meta.yml

```diff
@@ -0,0 +1,64 @@
+name: plink2_extract
+description: Subset plink pfiles with a text file of variant identifiers
+keywords:
+  - plink2
+  - extract
+tools:
+  - plink2:
+      description: |
+          Whole genome association analysis toolset, designed to perform a range
+          of basic, large-scale analyses in a computationally efficient manner
+      homepage: http://www.cog-genomics.org/plink/2.0/
+      documentation: http://www.cog-genomics.org/plink/2.0/general_usage
+      tool_dev_url: None
+      doi: ""
```

Comment:
Please add DOI from: `Chang CC, Chow CC, Tellier LCAM, Vattikuti S, Purcell SM, Lee JJ (2015)
Second-generation PLINK: rising to the challenge of larger and richer datasets.
GigaScience, 4.
`

(this paper is referenced in `plink2 --help`)

Reply:
7e782c0  :smile: